build_debian.sh

```diff
@@ -113,6 +113,12 @@ sudo cp files/apt/apt.conf.d/{81norecommends,apt-{clean,gzip-indexes,no-language
 ## Note: set lang to prevent locale warnings in your chroot
 sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y update
 sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y upgrade
+
+echo '[INFO] Install and setup eatmydata'
+sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install eatmydata
+sudo ln -s /usr/bin/eatmydata $FILESYSTEM_ROOT/usr/local/bin/dpkg
```

Comment:
>eatmydata [](http://example.com/codeflow?start=20&length=9)

Why create this symlink? If needed, install it in slave?

Reply:
The symlink is being created within the chroot for use within the chroot (the link is being created at `$FILESYSTEM_ROOT/usr/local/bin/dpkg` and pointing to `/usr/bin/eatmydata`). When chrooted, this allows all calls to `dpkg` to actually call `eatmydata`, which will then look for the actual binary in `PATH`.

Reply:
You mean something like below?
```
sudo  LANG=C chroot $FILESYSTEM_ROOT ln -s /usr/bin/eatmydata /usr/local/bin/dpkg
```

If yes, one minor comment: it is not obvious that the intention is `ln` inside chroot.

Reply:
Yes. I was trying to avoid the chroot here, but I'll change this to use chroot to make it clear.

Reply:
This line has now been changed to use `chroot` to make it clear that the symlink is meant for use inside the chroot.